manticore/native/cpu/arm.py

```diff
@@ -390,16 +390,16 @@ def __init__(self):
             "D30",
             "D31",
         ):
-            self._regs[reg_name] = Register(64)
+            self._registers[reg_name] = Register(64)
         # Flags
-        self._regs["APSR_N"] = Register(1)
-        self._regs["APSR_Z"] = Register(1)
-        self._regs["APSR_C"] = Register(1)
-        self._regs["APSR_V"] = Register(1)
-        self._regs["APSR_GE"] = Register(4)
+        self._registers["APSR_N"] = Register(1)
+        self._registers["APSR_Z"] = Register(1)
+        self._registers["APSR_C"] = Register(1)
+        self._registers["APSR_V"] = Register(1)
+        self._registers["APSR_GE"] = Register(4)
```

Comment:
At first I wasn't sure why you renamed this but then I caught the `_registers` in `aarch64.py`